src/test/java/uk/gov/hmcts/dts/fact/controllers/admin/list/AdminContactTypeControllerTest.java

```diff
@@ -0,0 +1,170 @@
+
+        final String contactTypeJson = OBJECT_MAPPER.writeValueAsString(contactType);
+
+        mockMvc.perform(put(BASE_PATH)
+                            .content(contactTypeJson)
+                            .contentType(MediaType.APPLICATION_JSON)
+                            .accept(MediaType.APPLICATION_JSON))
+            .andExpect(status().isNotFound());
+    }
+
+    @Test
+    void shouldCreateAndReturnNewContactType() throws Exception {
+        final ContactType contactType = CONTACT_TYPES.get(0);
+        when(adminContactTypeService.createContactType(contactType)).thenReturn(contactType);
+
+        final String contactTypeJson = OBJECT_MAPPER.writeValueAsString(contactType);
+
+        mockMvc.perform(post(BASE_PATH)
+                            .content(contactTypeJson)
+                            .contentType(MediaType.APPLICATION_JSON)
+                            .accept(MediaType.APPLICATION_JSON))
+            .andExpect(status().isCreated())
+            .andExpect(content().json(contactTypeJson));
+    }
+
+    @Test
+    void shouldReturnConflictIfContactTypeAlreadyExists() throws Exception {
+        final ContactType contactType = CONTACT_TYPES.get(0);
+        when(adminContactTypeService.createContactType(contactType)).thenThrow(DuplicatedListItemException.class);
+
+        final String contactTypeJson = OBJECT_MAPPER.writeValueAsString(contactType);
+
+        mockMvc.perform(post(BASE_PATH)
+                            .content(contactTypeJson)
+                            .contentType(MediaType.APPLICATION_JSON)
+                            .accept(MediaType.APPLICATION_JSON))
+            .andExpect(status().isConflict());
+    }
+
+    @Test
+    void shouldSuccessfullyDeleteContactType() throws Exception {
+        final Integer id = 200;
+        final String idJson = OBJECT_MAPPER.writeValueAsString(id);
+
+        mockMvc.perform(delete(BASE_PATH  + "/" + id)
+                            .content(OBJECT_MAPPER.writeValueAsString(idJson))
+                            .contentType(MediaType.APPLICATION_JSON)
+                            .accept(MediaType.APPLICATION_JSON))
+            .andExpect(status().isOk())
+            .andExpect(content().json(idJson));
+    }
+
+    @Test
+    void shouldReturnConflictIfContactTypeInUse() throws Exception {
+        final Integer id = 500;
+        doThrow(mock(ListItemInUseException.class)).when(adminContactTypeService).deleteContactType(id);
+
+        mockMvc.perform(delete(BASE_PATH  + "/" + id)
+                            .content(OBJECT_MAPPER.writeValueAsString(id))
+                            .contentType(MediaType.APPLICATION_JSON)
+                            .accept(MediaType.APPLICATION_JSON))
+            .andExpect(status().isConflict());
+    }
+
+    @Test
+    void deleteContactTypeShouldReturnNotFoundIfIdDoesNotExist() throws Exception {
+        final Integer id = 300;
```

Comment:
Not a super issue by any means, do these need to be different ID's each time? Could we define one and reuse it, since we are mocking out the service? : -  )

Reply:
have define ID variable to use across tests